content/articles/creating-a-hand-gesture-volume-controller-using-python-and-pycharm/index.md

```diff
@@ -0,0 +1,146 @@
+### **Creating A Hand Gesture Volume Controller Using Python And Pycharm**
+In the times we are living in, computers are everywhere. Whether it is in your place of work while listening to a presentation, your home while listening to your favorite music, or in entertainment places where DJs use them. They help us simplify some of our day-to-day activities. The problem is water, dust, and other things such as food spilling over them easily damage them. 
+
+At this point, a hand gesture volume controller would come in handy. You don't have to be concerned about the state of your hands. You just have to use your fingers and adjust the computer's volume to your level of satisfaction.
+
+### Introduction
+**Python** has libraries that will help us create our hand gesture volume controller. **Pycharm** is the IDE that we will use to run our python code. We will be using the **Pycharm** community version IDE. It is free to download and use from the internet. 
+
+In this tutorial, we will discuss the use of the python libraries we will need, how to use python to track hand gestures, and finally how to create a hand gesture volume controller using Python and Pycharm. A person using Windows, Linux, or Mac OS can follow through.
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Python libraries we will need](#python-libraries-we-will-need)
+- [Referral hand image](#referral-hand-image)
+- [Creating a hand gesture volume controller](#creating-a-hand-gesture-volume-controller)
+- [Results](#results)
+- [Conclusion](#Conclusion)
+
+### Prerequisites
+For you to follow through this article you need to:
+- Be familiar with the Python programming language.
+- Have **Pycharm** installed on their computer. If this is not the case, download it from here. [Pycharm Community Edition ](https://www.jetbrains.com/pycharm/download/)
+
+### Python libraries we will need
+The libraries that we will be interacting with while creating our hand gesture volume controller will be the following:
+- *numpy* This library will help us work with arrays.
+- *cv2* We will use this library to capture an image using the webcam and convert the image to RGB.
+- *mediapipe* This is an open-source library developed by Google. It is used for both face and gesture recognition. For the matter of this tutorial, we will use it for hand gesture recognition.
+- *math* We will use this library to find the distance between point number 4 (the thumb) and point number 8 (the index finger) using hypotenuse.
+- *ctypes* and *comtypes* These two libraries are depended upon by pycaw. Ctypes to provide C language compatible data types. Comtypes bases on the ctypes FFI(Foreign Function Interface) library.
+- *pycaw* We will need this library to access the device's speaker and its master volume.

```

Comment:
Instead of writing this section separately with the code, I'd rather you explain each library after its `pip install` equivalent.